src/satosa/frontends/oidcop/claims.py

```diff
@@ -0,0 +1,48 @@
+from collections import defaultdict
+
+
+def combine_return_input(values):
+    return values
+
+
+def combine_select_first_value(values):
+    return values[0]
+
+
+def combine_join_by_space(values):
+    return " ".join(values)
+
+
+combine_values_by_claim = defaultdict(
+    lambda: combine_return_input,
+    {
+        "sub": combine_select_first_value,
```

Comment:
better have this mapping in the frontend configuration, any other additional claim won't take any additional commit to source tree